main/README.md

```diff
@@ -4,53 +4,109 @@
+footer: Apache-2.0 Licensed | Copyright © 2023 - Agoric
 ---
 
-<div class="flex flex--column flex--center">
-  <p>
-    Learn about ERTP (Electronic Rights Transfer Protocol), a uniform way of transferring tokens and other digital assets in JavaScript.
-  </p>
-  <Button-Action-Link
-    text="Explore ERTP"
-    link="/guides/ertp/"
-  />
+<div class="home-banner">
+  Receive our latest tutorials, webinar invites, and much more! <a href="https://agoric.com/dev-newsletter" style="color: white;"">Join our newsletter</a>
 </div>
-<br>
-<div class="flex flex--column flex--center">
-  <p>Check out Zoe, a smart contract framework responsible for enforcing what we call "offer safety". The smart contracts that run on top of Zoe are responsible for figuring out proposed reallocations of resources.
-  </p>
-  <Button-Action-Link
-    text="Build on Zoe"
-    link="/guides/zoe/"
+
+<div class="home-section">
+  <HomeButtonHeader
+    title="Agoric documentation"
+    text="The Agoric platform makes it possible to write safer smart contracts with your JavaScript skillset."
+  />
+  <HomeButtonRow
+    title1="What's Agoric"
+    text1="Learn about Agoric and its Javascript platform"
+    link1="/guides/what-is-agoric/"
+    title2="Getting Started"
+    text2="Set up your environment and start building apps"
+    link2="/guides/getting-started/"
+    title3="Contract Framework"
+    text3="Set up your environment and start building"
+    link3="/guides/zoe/"
   />
+  <HomeButtonRow
+    title1="Token Standards"
+    text1="Understand the ins and outs of Agoric's ERTP standard"
+    link1="/guides/ertp/"
+    title2="Samples"
+    text2="Look through our smart contract and dapp examples"
+    link2="/guides/zoe/contracts/"
+    title3="Integrations"
+    text3="Browse through our catalogue of integrations"
+    link3="/guides/chainlink-integration.html#overview"
+    />
 </div>
 
-## The Platform
+<div class="home-section">
+  <HomeButtonHeader
+    title="Blockchain resources"
+    text="The tools your need to get the job done."
+  />
+  <HomeButtonRow
+    title1="Block Explorer"
+    text1="View transactions across the Agoric chain"
+    link1="https://bigdipper.live/agoric"
+    title2="Keplr Wallet"
+    text2="A native wallet running with HardenedJS under the hood"
+    link2="https://www.keplr.app/download"
+    title3="Component Library"
+    text3="Pre-built smart contracts for DeFi, NFTs, and cross-chain!"
+    link3="https://components.agoric.com/"
+  />
+  <HomeButtonRow
+    title1="Cosmos SDK"
+    text1="Our battle-tested consensus mechanism"
+    link1="https://docs.cosmos.network/"
+    title2="IBC"
+    text2="The protocol ensuring Agoric is interoperable with 60÷ chains"
+    link2="https://ibc.cosmos.network/main"
+    title3="Bounties"
+    text3="A rotating list of incentivized bounties to grow our platform"
+    link3="https://components.agoric.com/bounties/open-bounties"
```

Comment:
Placing "Bounties" here next to Cosmos and IBC seems odd.

I suggest swapping it with Integrations.

Reply:
Swapped